### PR TITLE
Fix bugs: create user status value and emails with the "ñ" letter

### PR DESCRIPTION
**Pull request by RamOwen99**
The user can now gave us the status field with the values null, 0 and 1. If we receive other value, we send a message error of what could happen in that field.
The users' emails now can have the "ñ" letter

## When the user don't type his first name and last name
imagen
imagen

## When the user type an invalid email
imagen

## When the user type an invalid password
imagen

## When the user's email it already exist in the database
imagen

### CODE REVIEW

**Review by ivanflores15 — APPROVED** [review]
Did you test REGEX to validate email?

File: src/main/java/com/agilethought/atsceapi/exception/ErrorMessage.java [hunk 1]
```diff
@@ -5,6 +5,10 @@
     public static final String BAD_REQUEST_MESSAGE = "The field ";
     public static final String BAD_REQUEST_MESSAGE_TYPE = "needs to be 1 or 2";
     public static final String BAD_REQUEST_MESSAGE_FIRST_NAME_LASTNAME = "is wrong because you need to type it";
-    public static final String BAD_REQUEST_MESSAGE_EMAIL_PASSWORD = "is wrong because it doesn't have the correct format";
```
Comment (luisgalindoluna) [inline src/main/java/com/agilethought/atsceapi/exception/ErrorMessage.java]: Your current changes are fine, but I want to take advantage of your current PR, to make a simple request. Do you think you can change those 2 messages? 
If I'm not mistaken those errors are sent when the user sends upper case letters and you only want them to input lowercase ones, correct? 
If that's the case I think you should try to be more clear, like "Name fields should be written with lower case letters only", for example. Same with email.

**Review by luisgalindoluna — CHANGES_REQUESTED** [review]
Have a look at my comment

File: src/main/java/com/agilethought/atsceapi/exception/ErrorMessage.java (hunk 1 above)
Comment (RamOwen99) [reply src/main/java/com/agilethought/atsceapi/exception/ErrorMessage.java]: Actually, we can receive letters in uppercase in the firstname and lastname, that's why I delete the method that check if the string is in lowercase.
We can receive uppercase letter in the email and password, that´s why I spare the constant 
`BAD_REQUEST_MESSAGE_EMAIL_PASSWORD` in two constants to have a better undestanding of what could faild

Comment (luisgalindoluna) [reply src/main/java/com/agilethought/atsceapi/exception/ErrorMessage.java]: I see, but in that case when do those 2 messages pop up then? When the user leaves those fields blank? 
My problem with those messages is that they aren't descriptive enough I don't know what they mean when I read them. So my request is "Change them to something more specific and easy to understand"

Comment (RamOwen99) [reply src/main/java/com/agilethought/atsceapi/exception/ErrorMessage.java]: I upload the messages, let me know if they are good or if I need to improve the messages

File: src/main/java/com/agilethought/atsceapi/exception/ErrorMessage.java [hunk 2]
```diff
@@ -4,7 +4,11 @@
 
     public static final String BAD_REQUEST_MESSAGE = "The field ";
     public static final String BAD_REQUEST_MESSAGE_TYPE = "needs to be 1 or 2";
-    public static final String BAD_REQUEST_MESSAGE_FIRST_NAME_LASTNAME = "is wrong because you need to type it";
-    public static final String BAD_REQUEST_MESSAGE_EMAIL_PASSWORD = "is wrong because it doesn't have the correct format";
+    public static final String BAD_REQUEST_MESSAGE_FIRST_NAME_LASTNAME = "is required and you don't type it";
```
Comment (luisgalindoluna) [inline src/main/java/com/agilethought/atsceapi/exception/ErrorMessage.java]: I hate to nitpick....but humor one last request of mine. Switch that "don't" to "didn't". So....
"is required and you didn't type it". I'm kinda stingy when it comes to english, sorry.

**Review by luisgalindoluna — CHANGES_REQUESTED** [review]

File: src/main/java/com/agilethought/atsceapi/exception/ErrorMessage.java (hunk 2 above)
Comment (RamOwen99) [reply src/main/java/com/agilethought/atsceapi/exception/ErrorMessage.java]: Ok, don't worry, I'll change that

**Review by luisgalindoluna — APPROVED** [review]
Approved. thank you

**Review by EmiliaGuillen-dev — APPROVED** [review]
looks good

**Pull request merged**